.circleci/config.yml

```diff
@@ -7,42 +7,25 @@ jobs:
       - checkout
       - setup_remote_docker
       - run: docker build .
-  node8:
+  node10:
     docker:
-      - image: node:8-alpine
+      - image: node:10-alpine
     steps:
       - run: apk add --no-cache make bash git ca-certificates
       - checkout
       - restore_cache:
           keys:
-          - node8-dependencies-{{ checksum "yarn.lock" }}
-          - node8-dependencies-
+          - node10-dependencies-{{ checksum "yarn.lock" }}
+          - node10-dependencies-
       - run: yarn install --frozen-lockfile
       - save_cache:
           paths:
             - node_modules
-          key: node8-dependencies-{{ checksum "yarn.lock" }}
-      - run: yarn test
```

Comment:
what about the developer who use node 8 and node 7??? this would not be compatible.